.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [16, 18]
+        node: [18, 20]
```

Comment:
Do we have a good reason to run tests with two different node versions? If not, I think we can directly go to v22 (latest LTS).